src/rust/dbv2/migrations/2023-12-01-173351_add_total_fees/up.sql

```diff
@@ -0,0 +1,36 @@
+-- Your SQL goes here
+DROP VIEW api.orders;
+
+
+ALTER TABLE aggregator.user_history
+ADD COLUMN total_fees_paid_in_quote_subunits NUMERIC(20,0);
+
+
+WITH total_fees AS (
+    SELECT
+        SUM(taker_quote_fees_paid) AS total_fees_paid_in_quote_subunits,
+        taker_order_id,
+        market_id
+    FROM fill_events AS f
+    WHERE f.taker_address = f.emit_address
+    GROUP BY market_id, taker_order_id
+)
+UPDATE aggregator.user_history AS u
+SET total_fees_paid_in_quote_subunits = f.total_fees_paid_in_quote_subunits
+FROM total_fees AS f
+WHERE f.taker_order_id = u.order_id
+AND f.market_id = u.market_id;
```

Comment:
After looking through the rest of the logic in this PR, it looks to me like this block was an attempt to backfill fee data, rather than restarting the DSS, during development. But given the other aggregator logic you're proposing, I think you should be able to cut this out

How about updating this migration to the simpler version below:

```sql
DROP VIEW api.orders;


ALTER TABLE aggregator.user_history
ADD COLUMN total_fees_paid_in_quote_subunits NUMERIC(20, 0) NOT NULL;


CREATE VIEW
  api.orders AS
SELECT
  *
FROM
  aggregator.user_history;


GRANT
SELECT
  ON api.orders TO web_anon;
```

Reply:
Yes, it is supposed to backfill.

I'm not sure about taking this out. We should strive to make the DSS have a better developer experience, and this is part of this.

Let's talk more about this on the daily.

Reply:
Noting here that backfill logic enables hot DSS upgrades, and that we need to also account for last indexed txn version number for the relevant pipeline when constructing hot upgrade migrations